packages/calypso-e2e/src/lib/components/editor-gutenberg-component.ts

```diff
@@ -45,7 +45,8 @@ export class EditorGutenbergComponent {
 	async resetSelectedBlock(): Promise< void > {
 		const editorCanvas = await this.editor.canvas();
 		const locator = editorCanvas.locator( selectors.title );
-		await locator.click();
+		// Every now and then, a block toolbar can cover the title, so we "force" the click by doing it via the event directly.
+		await locator.dispatchEvent( 'click' );
```

Comment:
What if, instead of selecting the `Title` block, we select the editor toolbar at the top?

This part:

![Screenshot 2023-10-13 at 4 06 28 PM](https://github.com/Automattic/wp-calypso/assets/6549265/dc4b7254-29d3-4d6e-9f87-a21f5f58d20d)

Reply:
Implemented and tried this method locally and it seems to work just fine.

I'd like to hear your thoughts on this; would "promoting" this method to the `EditorPage` be too much?

Reply:
Ahhh, I think I finally figured it out!

I did some more testing, and unfortunately, the solution you had was also failing at the same points as mine was.

The reason for both... the underlying issue is just focus! What determines block availability is what currently has focus. Dispatching a `click` event doesn't actually set focus. (As opposed to manually clicking which both dispatched `click` and sets DOM focus).

So, I rolled back to my original solution and also added `locator.focus()`, and it works like a charm! 🎉 

Thank you so much for the extra help investigating! 🙇